site/daemon.md

```diff
@@ -81,6 +81,7 @@ fluxd requires setup and offers customization though a multitude of flags.
 | --registry-ecr-region                            | `[]`                     | Allow these AWS regions when scanning images from ECR (multiple values allowed); defaults to the detected cluster region
 | --registry-ecr-include-id                        | `[]`                     | Include these AWS account ID(s) when scanning images in ECR (multiple values allowed); empty means allow all, unless excluded
 | --registry-ecr-exclude-id                        | `[<EKS SYSTEM ACCOUNT>]` | Exclude these AWS account ID(s) when scanning ECR (multiple values allowed); defaults to the EKS system account, so system images will not be scanned
+| --registry-ecr-mandatory                         | `false`                  | exit with an error if the AWS API (and thereby ECR) is not accessible. Useful for escalating misconfiguration or outages that might otherwise go undetected.
```

Comment:
This should be `--registry-require=`, right?

Reply:
Yar, missed all the dependent docs etc., when I changed how the arg worked.